pkg/version/version.go

```diff
@@ -40,3 +46,23 @@ func (u uat) RoundTrip(r *http.Request) (*http.Response, error) {
 	r.Header.Set("User-Agent", UserAgent)
 	return u.tr.RoundTrip(r)
 }
+
+// DumpVersion dumps the version information to the log.
+func DumpVersion() {
```

Comment:
this implementation needs to be robust , as it stands it is not. Plus it depends on a CLI command.. which I don't want us to have a dependency for this trade-off. I think a better more robust solution is needed here. I believe golang has some in built features to do this.

Reply:
It is using already using builtin https://github.com/guacsec/guac/blob/7c3b1b9188e868c1ce2c8c21793cd7e6894aa244/.goreleaser.yaml#L67-L69

If the built-in fails, we use the commit SHA and the present date and run the non-released version.

Let me know if you have a better strategy to solve this issue.